route/src/route_generator_worker.cpp

```diff
@@ -347,7 +347,8 @@ namespace route {
             state_msg.routeID = route_msg_.route_name;
             state_msg.cross_track = current_crosstrack_distance_;
             state_msg.down_track = current_downtrack_distance_;
-            state_msg.lanelet_downtrack = ll_downtrack_distance_;
+            state_msg.lanelet_downtrack = ll_downtrack_distance_;            
+            state_msg.state = this->rs_worker_.convertRouteStateToInt(this->rs_worker_.get_route_state());
```

Comment:
You should be able to just make this
```
state_msg.state = this->rs_worker_.get_route_state();
```

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/pull/818/commits/cb5fb9be75570720d340a2e0151aed275157cd56